src/AspNet.Security.OAuth.Typeform/TypeformAuthenticationOptions.cs

```diff
@@ -0,0 +1,32 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+
+namespace AspNet.Security.OAuth.Typeform;
+
+/// <summary>
+/// Defines a set of options used by <see cref="TypeformAuthenticationHandler"/>.
+/// </summary>
+public class TypeformAuthenticationOptions : OAuthOptions
+{
+    public TypeformAuthenticationOptions()
+    {
+        ClaimsIssuer = TypeformAuthenticationDefaults.Issuer;
+        CallbackPath = TypeformAuthenticationDefaults.CallbackPath;
+
+        AuthorizationEndpoint = TypeformAuthenticationDefaults.AuthorizationEndpoint;
+        TokenEndpoint = TypeformAuthenticationDefaults.TokenEndpoint;
+        UserInformationEndpoint = TypeformAuthenticationDefaults.UserInformationEndpoint;
+
+        Scope.Add("offline");
```

Comment:
Is this a necessary default?

Reply:
No, it is just a recommendation if we need a refresh token in response.